erpnext/utilities/doctype/video/video.json

```diff
@@ -15,7 +15,14 @@
   "publish_date",
   "duration",
   "section_break_7",
-  "description"
+  "description",
+  "image",
+  "youtube_tracking_section",
+  "like_count",
+  "view_count",
+  "col_break",
+  "dislike_count",
+  "comment_count"
```

Comment:
Can we have these above the description? Seems like it would do better up there instead.